lib/c_glib/test/testthrifttestzlibclient.cpp

```diff
@@ -107,8 +107,8 @@ class TestHandler : public ThriftTestIf {
     out = thing;
   }
 
-  std::string testUuid(const std::string thing) override {
-    cout << "[C -> C++] testUuid(\"" << std::hex << thing << "\")" << '\n';
+  apache::thrift::TUuid testUuid(const apache::thrift::TUuid thing) override {
```

Comment:
If you use const ref on the parameter and return value instead, it avoids copies.